### DISCUSSION TITLE
Does nuxt generate a manifest.json file?

### DISCUSSION THREAD

**Discussion by abdul-alhasany** [nuxt/nuxt#20772] [Questions]
Hi
I am trying to create a headless WordPress theme. Because nuxt produces html files when I run `nuxt generate` I can not include them in WordPress theme. 

Currently, I am adding the generated files to WordPress root folder (adjacent to wp-content, wp-admin .. etc) but it is causing some issues for vue-router. 

Does nuxt generate a `manfiest.json` file like vite when using `manifest: true`. I tried adding this to `nuxt.config.ts` but it did not produce anything.

**Comment by xdavidwu** [comment]
It does, but seems to be removed by af2835fa7f472177e3e103bdc03662785724022a from https://github.com/nuxt/framework/pull/7021
